sdk/documenttranslation/Azure.AI.Translation.Document/Azure.AI.Translation.Document.sln

```diff
@@ -3,11 +3,11 @@ Microsoft Visual Studio Solution File, Format Version 12.00
 # Visual Studio Version 16
 VisualStudioVersion = 16.0.30804.86
 MinimumVisualStudioVersion = 10.0.40219.1
-Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Azure.AI.Translator.DocumentTranslation", "src\Azure.AI.Translator.DocumentTranslation.csproj", "{6BE1651C-C3F6-4363-8717-FFF1D95A3643}"
+Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Azure.AI.Translation.Document", "src\Azure.AI.Translation.Document.csproj", "{6BE1651C-C3F6-4363-8717-FFF1D95A3643}"
```

Comment:
This doesn't follow naming with other packages. I realize this is the latest discussion, but I've replied to the email. Following suite with other libraries, this should be `Azure.AI.Translation.Documents`.

Reply:
It is an in-between and is prioritizing service consistency (Text and Document singular). 
Let me know if you consider this a blocker

Reply:
Not a blocker, but with the name switch-around from "Document Translator" (or even "Document Translation") to "Translation.Document", the product name is lost. These packages and namespaces are conceptual. Given the guidelines, if anything the service name is in the second or third position (most often the third, as the second position is a topic, like "AI" or "Security".

This is no different than Blob Storage becoming Storage.Blobs.

/cc @johanste @KrzysztofCwalina

Reply:
That said, this is not a problem to be solved and hold up this PR, so I'll sign off.

Reply:
Although it is true the service name is `Translator`, `Translation` is the user-friendly word the service has been using for comunication. 
Also, service has given us green light with this name.

For the upcoming release of `1.0.0-beta.1` we are going to go with `Azure.AI.Translation.Document` if, after more conversations, the name needs to change, although not ideal, we can definitely do it.